api/v1/sriovoperatorconfig_types.go

```diff
@@ -45,6 +45,8 @@ type SriovOperatorConfigSpec struct {
 	ConfigurationMode ConfigurationModeType `json:"configurationMode,omitempty"`
 	// Flag to enable Container Device Interface mode for SR-IOV Network Device Plugin
 	UseCDI bool `json:"useCDI,omitempty"`
+	// DisablePlugins is a list of sriov-network-config-daemon plugins to disable
+	DisablePlugins []string `json:"disablePlugins,omitempty"`
```

Comment:
you can add a validation at the api level

	// +kubebuilder:validation:Enum={"mellanox"}

Reply:
ack. thx

Reply:
tried this out, it doesnt work for lists just for scalar values

`spec.disablePlugins: Unsupported value: []interface {}{"mellanox"}: supported values: "mellanox"`

Reply:
I see, perhaps this https://github.com/kubernetes-sigs/kubebuilder/issues/2812?

Reply:
works. PR updated.